content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -452,6 +453,32 @@ interval. In the above example, all desired states from the past 5 minutes will
 This approximates a rolling maximum, and avoids having the scaling algorithm frequently
 remove Pods only to trigger recreating an equivalent Pod just moments later.
 
+### Tolerance {#tolerance}
+
+{{< feature-state feature_gate_name="HPAConfigurableTolerance" >}}
+
+The `tolerance` field configures a threshold for metric variations, preventing the
+autoscaler from scaling for changes below that value.
+
+This tolerance is defined as the amount of variation around the desired metric value under
+which no scaling will occur. For example, consider a HorizontalPodAutoscaler configured
+with a target memory consumption of 100Mi and a scale-up tolerance of 5%:
+
+```yaml
+behavior:
+  scaleUp:
+    tolerance: 0.05 # 5% tolerance for scale up
+```
+
+With this configuration, the HPA algorithm will only consider scaling up if the memory
+consumption is higher than 105Mi (i.e. 5% above the target).
+
+If you don't set this field, the HPA applies the default cluster-wide tolerance of 10%. This
+default can be updated for both scale-up and scale-down using the
+[kube-controller-manager](/docs/reference/command-line-tools-reference/kube-controller-manager/)
+`--horizontal-pod-autoscaler-tolerance` flag. (You can't use the Kubernetes API
```

Comment:
```suggestion
`--horizontal-pod-autoscaler-tolerance` command line argument. (You can't use the Kubernetes API
```
Outside of Golang, "flag" means a boolean-valued argument. I recommend not using "flag" to mean command line arguments that aren't booleans.